### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by noahtalerman**
- Remove the "why"
  - @noahtalerman: Hypothesis is that users who land here want to get started fast. They already know the why or don't care
- Remove note about `fleetctl gitops`. Users don't interact w/ it
- Simplify headers

### CODE REVIEW

**Comment by noahtalerman** [comment]
> Remove the "why"

FYI @rachaelshaw I took this TODO

**Comment by noahtalerman** [comment]
Hey @getvictor unrelated to this PR, what do we use this Fleet GitOps repo in GitLab for? https://gitlab.com/developers6254743/fleet-gitops

I was recently invited.

**Comment by getvictor** [comment]
> Hey @getvictor unrelated to this PR, what do we use this Fleet GitOps repo in GitLab for? https://gitlab.com/developers6254743/fleet-gitops
> 
> I was recently invited.

We need a GitLab repo to test the GitLab CI (.gitlab-ci.yml). It is a mirror of this repo. But we need the URL to be something like https://gitlab.com/fleetdm/fleet-gitops
I'm waiting for @lukeheath to sort it out.

**Comment by noahtalerman** [comment]
> We need a GitLab repo to test the GitLab CI (.gitlab-ci.yml).

@getvictor ah, ok.

Will our users ever interact w/ it? Or is the best practice to clone the GitHub repo using GitLab?

Wondering b/c, as someone new to GitLab, I'm confused looking at the steps in the README:

Screenshot 2024-10-29 at 4 21 20 PM
Screenshot 2024-10-29 at 4 22 09 PM

How do I clone the repo if I'm using GitLab? Do I create a GitLab repo after I clone the GitHUb repo?

**Comment by getvictor** [comment]
> > We need a GitLab repo to test the GitLab CI (.gitlab-ci.yml).
> 
> @getvictor ah, ok.
> 
> Will our users ever interact w/ it? Or is the best practice to clone the GitHub repo using GitLab?
> 
> Wondering b/c, as someone new to GitLab, I'm confused looking at the steps in the README:
> 
> Screenshot 2024-10-29 at 4 21 20 PM Screenshot 2024-10-29 at 4 22 09 PM
> 
> How do I clone the repo if I'm using GitLab? Do I create a GitLab repo after I clone the GitHUb repo?

Both GitHub and GitLab use git. So terms like `clone` and `push` are git-specific, and don't refer to the git UI. If the repo is mirrored in both GitHub and GitLab, then users can clone from either place.

**Comment by lukeheath** [comment]
@noahtalerman @getvictor I created it: https://gitlab.com/fleetdm/fleet-gitops

@getvictor You should have access but let me know if not.

**Comment by noahtalerman** [comment]
> If the repo is mirrored in both GitHub and GitLab, then users can clone from either place.

@getvictor does this mean that any change (commit) in GitHub will be reflected in GitLab and vice versa?

@getvictor I took a pass at simplifying the setup instructions. What do you think?


For later:

Now that we have a GitHub and [GitLab repo](https://gitlab.com/fleetdm/fleet-gitops), my understanding is that we could simplify the setup steps further by removing these steps:

Screenshot 2024-11-05 at 12 57 07 PM

Screenshot 2024-11-05 at 12 57 52 PM

I think we can come back to this later. My understanding is that this would require making changes to the GitHub and GitLab repos and maintaining these repos separately.

File: README.md [hunk 1]
```diff
@@ -45,13 +17,11 @@ Rather than relying on multiple teams for tools like Intune or Jamf, build a uni
...
-5. Push your cloned and modified code to your repo.
-
-6. In GitHub, enable the `Apply latest configuration to Fleet` GitHub Actions workflow, and run workflow manually. Now, when anyone pushes a new commit to the default branch, the action will run and update Fleet. For pull requests, the workflow will do a dry run only.
+5. In GitHub, enable the `Apply latest configuration to Fleet` GitHub Actions workflow, and run workflow manually. Now, when anyone pushes a new commit to the default branch, the action will run and update Fleet. For pull requests, the workflow will do a dry run only.
 
-## Set up on GitLab
+## GitLab setup
 
-1. Create your own GitLab repository.
+1. Clone the [GitLab repository](https://gitlab.com/fleetdm/fleet-gitops), create your own GitLab repository, and push your clone to your new repo.
```
Comment (getvictor) [inline README.md:24]: At this point, the pipeline/action will run and fail because the secrets are not present. We could call it out.

**Review by getvictor — APPROVED** [review]
Looks good. One comment.

File: README.md (hunk 1 above)
Comment (noahtalerman) [reply README.md]: > he pipeline/action will run and fail because the secrets are not present. 

@getvictor only for GitLab right? I added a note.

**Comment by noahtalerman** [comment]
> The deletion step 4 is optional, so we could simply remove it.

Removed it 👍

File: README.md (hunk 1 above)
Comment (getvictor) [reply README.md]: For both (although I haven't tried this updated flow with GitLab)

**Comment by noahtalerman** [comment]
Decided to merge this.

If we learn that the new instructions don't work we can quickly update.

**Pull request merged**